.metalinter.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Linters": {
+    "unused": {
+      "Command": "unused -tags 'big'",
+      "Pattern": "PATH:LINE:COL:MESSAGE"
+    },
+    "gosimple": {
+      "Command": "gosimple -tags 'big'",
+      "Pattern": "PATH:LINE:COL:MESSAGE"
+    }
+  },
+  "Enable": [
+    "deadcode",
+    "varcheck",
+    "structcheck",
+    "goconst",
+    "ineffassign",
+    "unconvert",
+    "misspell",
+    "golint",
+    "gosimple",
+    "unused",
+    "maligned"
```

Comment:
Is it worth adding go vet and go fmt as well?

Reply:
opened https://github.com/m3db/m3ninx/issues/27 to track